pandas/core/frame.py

```diff
@@ -4915,6 +4915,10 @@ def assign(self, **kwargs) -> DataFrame:
             A new DataFrame with the new columns in addition to
             all the existing columns.
 
+        See Also
+        -------
```

Comment:
```suggestion
        --------
```
There should be a hyphen under each character in "See Also" (so 8 in total)

Reply:
Oh I see, thank you for pointing that out!